ehitchhiking-ui/src/app/pages/main-screen/main-screen-component/main-screen.component.sass

```diff
@@ -17,5 +18,17 @@
   height: 100%
   padding: 0
   margin: 0
-
-
+.green
+  background-color: $green
+.red
+  background-color: $red
+.save-route
```

Comment:
pls set `position: fixed` for `main-screen-action-btn`. There is no reason to set the same styles for two buttons

Reply:
Is `z-index: 1` still required ?

Reply:
fixed
no